src/main/java/com/jdig/model/SorbsBlacklists.java

```diff
@@ -0,0 +1,253 @@
+package com.jdig.model;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Enumeration of all services provided by SORBS. 
+ * @see <a href="http://www.sorbs.net/using.shtml">SORBS site</a>
+ */
+public enum SorbsBlacklists {
```

Comment:
Since you created a generic CSV file, I guess we can have a generic Enum for all Blacklists.